kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -2437,7 +2437,7 @@ button.kb-data-obj {
 .kb-data-list-add-data-button {
 
     cursor:pointer;
-    background-color: #F44336;
+    background-color: #4379B1;
```

Comment:
Some thoughts -- if we are likely to be make a number of colour changes, we should consider creating a ticket to refactor this and the other css files to use SASS instead. There are (or were, last time I looked at it) a load of errors in the css file, and an initial version of the SASS file can be generated from the CSS using a tool. Having all the colour variables together in a single file would enable easier colour scheme changes.

Reply:
+1000

Reply:
@ialarmedalien that works for me. Can you put a ticket into the backlog for it?

Reply:
Done! https://kbase-jira.atlassian.net/browse/DATAUP-246